atlp-rwanda/e-commerce-ones-and-zeroes-fn
# [Delivered #187355059]Users should be able to view and edit their pro…

## 1.What does this PR do?
Users should be able to view and edit their profiles + billing address
# ⚠️⚠️How to test this PR
1. Clone `develop` branch and checkout to `user-profile-settings2`
2. Install all required dependencies `npm install`
3. Follow `.env.example` to create your `.env` file
4. Start the application using `npm start`
5.  Navigate to login and start by signing up, verify your account 
  `N.B:`  during verification you will not see grafical UI but after clicking button to verify check if you recieve another email for successfully verification and go ba back to your application try to login
6. After login navigate to `profile` link in menu and update your personal information and billing address

## 2.Description
I want to get my profile created once I register for the application.
I want my profile to display my:
 ### 1.Personal Information
   -name, email,gender, birthdate,
   -preferred language, preferred currency,where you live,
 ### 2.Billing Address (check the right fields of the billing address)

![Screenshot 2024-06-24 114309](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/57124925/5c091a61-b502-447e-bf3b-cd912052e5f2)
![Screenshot 2024-06-24 114343](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/57124925/fef6fb93-cbab-47ba-b233-8e4ff4a81b19)

## 3.Acceptance Criteria
I want to be able to Edit my profile, but I should NOT be able to edit another users profile
### Scenario:
Given User registers to be a member of the App.
Registration is done.
User Profile is created.
